### PR TITLE
Fix awscli tests in all situations

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: run/core/awscli/test.sh [hunk 1]
```diff
@@ -214,6 +216,15 @@ function test_lookup_object_prefix() {
         out="${bucket_name}"
```
Comment (kannappanr) [inline run/core/awscli/test.sh]: You could be at this point because of `Put Object` failure, not necessarily because of `make bucket` failure.

**Comment by harshavardhana** [comment]
Done fixed @kannappanr

File: run/core/awscli/test.sh [hunk 2]
```diff
@@ -241,21 +245,18 @@ function test_lookup_object_prefix() {
...
         rv=$?
-    fi
-
-    # Request should fail.
-    if [ $rv -eq 0 ]; then
-        # clean up and log error
-        ${AWS} s3 rb s3://"${bucket_name}" --force > /dev/null 2>&1
-        log_failure "$(get_duration "$start_time")" "${function}" "${out}"
-        return 1
+        if [ $rv -ne 255 ]; then
```
Comment (kannappanr) [inline run/core/awscli/test.sh]: When tested against `AWS S3`, this is returning 0

File: run/core/awscli/test.sh [hunk 3]
```diff
@@ -205,59 +205,46 @@ function test_lookup_object_prefix() {
...
 
-    # Request should fail.
-    if [ $rv -eq 0 ]; then
-        # clean up and log error
-        ${AWS} s3 rb s3://"${bucket_name}" --force > /dev/null 2>&1
-        log_failure "$(get_duration "$start_time")" "${function}" "${out}"
-        return 1
-    fi
-
     if [ $rv -ne 0 ]; then
```
[the remark this answers is not in the document]
Comment (harshavardhana) [reply run/core/awscli/test.sh]: no @kannappanr rv = 0 in the previous test that is on head-object on datafile-1-MB so the check is correct.

**Comment by nitisht** [comment]
Travis seems to fail because of #294 , taking this in.

**Pull request merged**